arkade5/start.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+# set -x
+echo "Archive bucket:       $ARCHIVE_BUCKET"
+echo "Archive obj_id:       $ARKIVUTTREKK_OBJ_ID"
+echo "Archive type:         $ARKIV_TYPE"
+echo "Azure Storage Account $AZURE_STORAGE_ACCOUNT"
+echo "Arkade output report  $OUTPUT_PATH_LOG"
+
+STORE="/objectstore"
+TARGET="$STORE/$ARKIVUTTREKK_OBJ_ID/content"
+CONTAINER="$ARCHIVE_BUCKET"
+
+
+mkdir -p /opt/output
+mkdir -p $STORE
+
+/usr/local/bin/goofys "wasb://${CONTAINER}@${AZURE_STORAGE_ACCOUNT}.blob.core.windows.net" "$STORE"
+
+dotnet /arkade5/Arkivverket.Arkade.CLI.dll \
+    test \
+    -a "$TARGET" \
+    -p /tmp -o /opt/output  \
+    -t "$ARKIV_TYPE" > /opt/output/arkade_output
```

Comment:
Tanker rundt denne endringen?
Gjør det litt enklere å forstå hvilke CLI flagg som brukes
```suggestion
dotnet /arkade5/Arkivverket.Arkade.CLI.dll \
    test \
    --type $ARKIV_TYPE \
    --archive $TARGET \
    --processing-area /tmp \
    --output-directory /opt/output > /opt/output/arkade_output
```